Lib/diffenator/diff.py

```diff
@@ -414,30 +436,18 @@ def _diff_images(img_before, img_after):
     pixels.
 
     TODO (M FOLEY) Crop images so there are no sidebearings to glyphs"""
-    width_before, height_before = img_before.size
-    width_after, height_after = img_after.size
-    data_before = img_before.getdata()
-    data_after = img_after.getdata()
-
-    width, height = max(width_before, width_after), max(height_before, height_after)
-    offset_ax = (width - width_before) // 2
-    offset_ay = (height - height_before) // 2
-    offset_bx = (width - width_after) // 2
-    offset_by = (height - height_after) // 2
+    pixels1 = img_before.convert('L').getdata()
+    pixels2 = img_after.convert('L').getdata()
+    pixels = zip(pixels1, pixels2)
```

Comment:
This won't work for images which have different dimensions. Zip's iterator stops when the shortest input iterable is exhausted.

```
>>> a = [1,2,3]
>>> b = [1,2,3,4,5]
>>> zip(a, b)
[(1, 1), (2, 2), (3, 3)]
```

Reply:
there's https://docs.python.org/3/library/itertools.html#itertools.zip_longest

Reply:
That's a great suggestion but it still don't work in this case.

`PIL.Image.convert("L").getdata()` outputs an image's pixels as a single list, `[255, 0, 255...]`. Simply filling the missing values from the smallest list doesn't take into account the image dimensions which is crucial for selecting the correct pixel. In the old code, we use two for loops to get the correct x,y location, we then use these to get the correct index in the list.

Reply:
I restored the original image diff code!